### PR TITLE
1.2.0 opentdf release

### PR DESCRIPTION
**Pull request by cassandrabailey293**
Version bumps for OpenTDF 1.2.0 release.

See https://github.com/opentdf/opentdf/pull/168 for a similar pr for last release.

### CODE REVIEW

**Comment by cassandrabailey293** [comment]
https://github.com/opentdf/tests/actions

the tests with 1.2.0 frontend and 1.2.1 backend have passed, as well as the CI. can I get an approval?

File: quickstart/start.sh [hunk 1]
```diff
@@ -41,7 +41,7 @@ REWRITE_HOSTNAME=1
 # NOTE: 1.1.0 default values. When releasing a new version, move these below to
 # the api-version selector and update the default.
 services=(abacus attributes entitlement-pdp entitlement-store entitlements entity-resolution kas keycloak keycloak-bootstrap)
-chart_tags=(1.1.1{,,,,,,,})
+chart_tags=(1.2.1{,,,,,,,})
```
Comment (pflynn-virtru) [inline quickstart/start.sh:44]: wil this cause problems with different versions front 1.2.0 and back 1.2.1

**Review by pflynn-virtru — APPROVED** [review]

File: quickstart/start.sh (hunk 1 above)
Comment (cassandrabailey293) [reply quickstart/start.sh:44]: good point - maybe we need to put abacus on it's own here

Comment (cassandrabailey293) [reply quickstart/start.sh:44]: the offline-start-test did pass though..

Comment (cassandrabailey293) [reply quickstart/start.sh]: need to improve the test, but i fixed this line

**Pull request merged**